mappings/net/minecraft/client/player/EntityPlayerClient.mapping

```diff
@@ -5,6 +5,7 @@ CLASS none/boe net/minecraft/client/player/EntityPlayerClient
 	METHOD a swingHand (Lnone/qz;)V
 		ARG 0 hand
 	METHOD c getBlockPos ()Lnone/cn;
+	METHOD c setCurrentHand (Lnone/qz;)V
```

Comment:
Are you sure it's not just a client-side hand? I remember having problems with all the "Hand" function/variable confusions with 1.9 and Charset...